source/_components/sensor.repetier.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+title: Repetier-Server Sensor
+description: "Instructions how to add Repetier-Server sensors to Home Assistant."
+date: 2018-12-02 22:12
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Sensor
+ha_release: 0.84
+ha_iot_class: "Local Polling"
+---
+
+This component creates one sensor for each devices in your Repetier-Server installation.
+Each sensor contains information about the current state of the device, including job name, start time and percentage done.
+
+To enable this sensor, add the following lines to your `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: repetier
+    url: http://IP_ADDRESS
+    port: PORT
+    api_key: YOUR_API_KEY
+```
+
+Configuration options for the Repetier-Server Sensor:
+
+- **url** (*Required*): The URL to connect to without the port part.
```

Comment:
Please use our new style for defining configuration options :)
<https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration>

Reply:
Done - I think